src/IPNetwork.php

```diff
@@ -380,9 +380,11 @@ public function prepareInput($input)
             $sameNetworks = self::searchNetworks("equals", $params, $entities_id, false);
            // Check unicity !
             if ($sameNetworks && (count($sameNetworks) > 0)) {
-                return ['error' => __('Network already defined in visible entities'),
-                    'input' => false
-                ];
+                $return = ['input' => false];
+                if (!isset($input['_no_message'])) {
```

Comment:
You should not change the result format here, but you should instead update the `if (isset($preparedInput['error'])) {` to `if (isset($preparedInput['error']) && !isset($input['_no_message'])) {`